inference-engine/src/gna_plugin/transformations/decompose_mvn.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace GNAPluginNS {
+
+/**
+ * @brief Decompose MVN operation
+ */
+class DecomposeMVN : public ngraph::pass::MatcherPass {
```

Comment:
@sirzabek could you please add the detailed description of the transformation in the header? It would be great to provide some examples. Please take a looks at the example - https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/gna_plugin/transformations/broadcast_const.hpp

Reply:
I will add a link to official OV MVN layer documentation. This should be enough, because the transformation basically implements the formula given in the documentation. It wouldn't be feasible to create drawing of a graph transformation in the header file. I'll do it in some other PR not to trigger the CI pipeline again here.